src/components/markdownEditor/view/formats/applyWebLinkFormat.js

```diff
@@ -40,7 +40,7 @@ export default ({ getState, item, setState, isImage = null }) => {
       };
     }
   }
-  setState({ text: newText }, () => {
-    setState({ newSelection });
-  });
+
+  await setState({ text: newText, textUpdated: true });
+  await setState({ newSelection });
```

Comment:
await setState({ text: newText }, () => {
 await setState({ newSelection });
});

Reply:
its looks better right?

Reply:
I am not sure is it better or same but I can change with your recommended